.github/workflows/release.yml

```diff
@@ -26,11 +26,18 @@ jobs:
       # Setup .npmrc file to publish to npm
       - uses: actions/setup-node@v4
         with:
-          node-version: '20.x'
-          registry-url: 'https://registry.npmjs.org'
+          node-version: "20.x"
+          registry-url: "https://registry.npmjs.org"
       - run: yarn
       - run: yarn prepare
-      - run: yarn npm publish --access public 
+      - run: yarn npm publish --access public
         working-directory: packages/react-native-client
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  publish-android-client:
+    # actually it just warmup jitpack cache, so android client is available sooner
```

Comment:
**question:** 
What does this comment mean?

Reply:
>If i understand properly it tries to fetch released version from jitpack and this triggers build so it is available faster?

That is correct
 
>What does this comment mean?

This means that this is not `publish` action, but just ask jitpack to build/cache package. I will think on how to phrase it better

Reply:
I think that sth like this:
```
Packages built by Jitpack are not immediately available upon the creation of a tag or branch. Instead, availability occurs when an initial attempt is made to download the specified tag or branch. This mechanism ensures that Jitpack only builds a package when it is first requested. This approach efficiently handles the generation of newly released versions.
```

Reply:
I've updated comment